lib/updatehub/updatehub.c

```diff
@@ -54,6 +54,74 @@ LOG_MODULE_REGISTER(updatehub);
 #define UPDATEHUB_SERVER "coap.updatehub.io"
 #endif
 
+/* please refer to coap.c */
+
+#define GET_NUM_2(v) ((v) >> 4)
+
+static int attempts_download;
```

Comment:
Could we place these newly introduced variable here and below to `ctx` struct?

Reply:
moved